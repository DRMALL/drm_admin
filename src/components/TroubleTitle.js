



import React from 'react'
import PageTip from './PageTip'
import ContentLink from './ContentLink'
import { Link } from 'react-router'


const TroubleTitle = ({ head }) => (
  <div className='trouble-title' >
    <PageTip tip={ head.title } />
    <div className='trouble-title-right' >
      <Link to='/trouble/kind' className='trouble-title-kind' >分类管理</Link>
      <ContentLink tip = { head.tip } path={ head.path } />
    </div>
  </div>
  )

export default TroubleTitle

