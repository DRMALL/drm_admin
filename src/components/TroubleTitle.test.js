

import TroubleTitle from './TroubleTitle'
import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'

it('TroubleTitle test', () => {
  const head ={ title: '新增设备', tip: 'tip', path: '/machine/new' }
  const wrapper = render(<TroubleTitle head={ head } />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
