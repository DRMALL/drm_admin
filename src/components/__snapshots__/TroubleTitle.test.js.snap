// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TroubleTitle test 1`] = `
<div
  class="trouble-title"
>
  <div
    class="page-tip"
  >
    新增设备
  </div>
  <div
    class="trouble-title-right"
  >
    <a
      class="trouble-title-kind"
    >
      分类管理
    </a>
    <a
      class="content-link"
    >
      <div
        class="content-link-block"
      >
        <i
          class="anticon anticon-plus"
        />
      </div>
      <div
        class="content-link-text"
      >
        tip
      </div>
    </a>
  </div>
</div>
`;
